packages/relay/test/login.test.ts

```diff
@@ -349,3 +386,10 @@ function prepareUserLoginTwitterApiMock(
             },
         })
 }
+
+function getCurrentEpoch(control?: string) {
+    if (!control) return null
+    const binary = BigInt(control) >> BigInt(160)
+    const mask = (BigInt(1) << BigInt(48)) - BigInt(1)
+    return binary & mask
+}
```

Comment:
the same as epoch
there is a epoch signal in signupProof or any other proof
https://developer.unirep.io/docs/circuits-api/signup-proof#epoch